src/system/dev/vfs.c

```diff
@@ -89,19 +89,29 @@ int vfs_update_entry(int file, struct fs_driver const* const driver, void* arg)
 	return 0;
 }
 
-int _open(const char* file, int flags, int mode) {
-	struct _reent* r = _REENT;
-	// Check if the filename is too long or not NULL terminated
+bool file_size(const char* file) {
+	// Returns true if is under size constraint
 	size_t i = 0;
 	for (i = 0; i < MAX_FILELEN; i++) {
 		if (file[i] == '\0') {
 			break;
 		}
 	}
 	if (i == MAX_FILELEN) {
+		return false;
```

Comment:
return i != MAX_FILELEN;

Reply:
why not just return false ?

Reply:
The suggestion is to also get rid of the if statement. It's needlessly verbose.

Also:
Traditionally (i.e. prior to C99), there was no formal Boolean type in C. It doesn't really matter in this case, but since we can make the code more concise while also avoiding having an explicit `true` floating around for no reason, it might as well be done